lib/index.d.ts

```diff
@@ -129,3 +131,7 @@ export class DetailedError extends Error {
   originalResponse: HttpResponse
   causingError: Error
 }
+
+export interface Checksum {
```

Comment:
Once we have finalized the interface and options, we should also add documentation. But that can wait for now.